NitroxLauncher/App.xaml.cs

```diff
@@ -21,6 +24,14 @@ protected override void OnStartup(StartupEventArgs e)
                 DefaultValue = FindResource(typeof(Page))
             });
             
+            // Error if running from a temporary directory because Nitrox Launcher won't be able to write files directly to zip/rar
+            // Tools like WinRAR do this to support running EXE files while it's still zipped.
+            if (Directory.GetCurrentDirectory().StartsWith(Path.GetTempPath(), StringComparison.OrdinalIgnoreCase))
+            {
+                MessageBox.Show("Nitrox launcher should not be executed from a temporary directory. Install Nitrox launcher properly by extracting and moving it to a dedicated location on your PC.", "Invalid working directory", MessageBoxButton.OK, MessageBoxImage.Error);
```

Comment:
let's say , "by extracting ALL files" so someone doesn't just extract the launcher.